test/fixtures/company.fixtures.ts

```diff
@@ -0,0 +1,61 @@
+import { Accounts, CompanyProfile, ConfirmationStatement, RegisteredOfficeAddress } from 'ch-sdk-node/dist/services/company-profile/types'
```

Comment:
Fixtures are reusable objects that can be called are reused throughout many test files. Think of them as example test data.

Unit test files can override the individual properties on the fixture objects to target certain scenarios.